### ISSUE TITLE
Consolidate WMG dependencies

### ISSUE THREAD

**Issue by ebezzi** [chanzuckerberg/single-cell-data-portal#2393]
The WMG pipeline has a separate `backend/wmg/requirements.txt` which contains duplicate libraries. We should try to include the main `requirements.txt` and just leave the libraries that are specific to the pipeline (e.g. `pronto`). The same pattern is being used by `backend/corpora/api_server/requirements.txt`

Note: the WMG pipeline was finicky with respect to dependencies, so expect that just using the main `requirements.txt` will not work out of the box. If this is too complicated, we could create a `requirements-core.txt` with all the libraries that we know are common (numpy, numba, tiledb, pandas etc) and include it in both requirements.txt files.

**Comment by ashin-czi** [comment]
Ticket still in progress

Madison and I tried to merge a change to test out in Dev but seems like the deployment failed due to an OOM error:
![image.png](https://images.zenhubusercontent.com/62dee801e47e5625ea12d27f/0d38e426-0096-49b5-b238-258a4ee5c96a)

**Comment by ashin-czi** [comment]
Merged another PR but waiting on confirmation to see if it's working fine in Dev

**Comment by ashin-czi** [comment]
Seems like scExpression works fine in Dev with these changes from playing around with different tissues and filters. But might want to see if anyone else could do a more thorough run through of the UI to ensure everything still works
